packages/@aws-cdk/aws-certificatemanager/lib/dns-validated-certificate.ts

```diff
@@ -61,7 +61,7 @@ export class DnsValidatedCertificate extends cdk.Resource implements ICertificat
         const requestorFunction = new lambda.Function(this, 'CertificateRequestorFunction', {
             code: lambda.Code.fromAsset(path.resolve(__dirname, '..', 'lambda-packages', 'dns_validated_certificate_handler', 'lib')),
             handler: 'index.certificateRequestHandler',
-            runtime: lambda.Runtime.NODEJS_10_X,
+            runtime: lambda.Runtime.NODEJS_12_X,
```

Comment:
Why this change?

Reply:
Updated nodejs version to the most recent runtime, since this is the only 10.x function in my stacks now. But yes, irrelevant to the PR, side fix.

Reply:
Can you bring it back to `NODEJS_10_X` then? I will also accept a property that allows you to set it to `NODEJS_12_X` (but the default should be `NODEJS_10_X`) 🙂.